chokuretsu/guide/play-game.html

```diff
@@ -0,0 +1,222 @@
+    <title>The Series of Haruhi Suzumiya - Setup Guide</title>
+    <link rel="stylesheet" href="../style.css">
+
+    <!-- Loads the footer and sidebar -->
+    <script src="//code.jquery.com/jquery-1.10.2.js"></script>
+    <script>
+        $(function () {
+            $("footer").load("../footer.html");
+            $(".column-with-sidebar-bar").load("sidebar.html");
+        });
+    </script>
+
+    <!-- Web icons -->
+    <link rel="manifest" href="../manifest.json"/>
+    <link rel="shortcut icon" href="../assets/icons/favicon.ico" type="image/png" sizes="16x16"/>
+    <link rel="shortcut icon" href="../assets/icons/android-chrome-192x192.png" type="image/png" sizes="192x192"/>
+
+    <!-- Google fonts (for Nunito font) -->
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Nunito">
+
+    <!-- Font awesome (for icons) -->
+    <script src="https://kit.fontawesome.com/aa4c023dda.js" crossorigin="anonymous"></script>
+
+    <!-- Platform detection script -->
+    <script src="guide.js" type="text/javascript"></script>
+</head>
+<body>
+<main id="column-with-sidebar">
+    <meta id="redirect-if-device-not-selected">
+    <div class="column-with-sidebar-body">
+        <div class="one-column-text" id="game-logo-container">
+            <object class="image-graphic drop-shadow" width="100%" data="../assets/game-logo.svg" type="image/svg+xml">
+                <img src="../assets/game-logo.png" alt="The Series of Haruhi Suzumiya logo"/>
+            </object>
+        </div>
+
+        <a href="choose-platform.html">
+            <div class="guide-platform-indicator hidden drop-shadow">
+                <object id="guide-platform-indicator-icon" width="20em" type="image/svg+xml"></object>
+                <b id="guide-platform-indicator-text"></b>
+            </div>
+        </a>
+
+        <div class="one-column-text">
+            <h1>Playing the game</h1>
+            <p>Now that you've got the patched ROM file, it's time to get ready to start playing!</p>
+        </div>
+
+        <!-- Playing on an emulator -->
+        <div class="one-column-text hidden emu-show">
+            <h3>Emulating the game</h3>
+            <p>There are a few options for DS emulators, but the one we recommend is MelonDS for its accuracy,
+                performance and feature set.</p>
+            <p>If you own a Wii U console, Nintendo's official Virtual Console DS emulator provides accurate emulation,
+                complimented nicely by the Wii U Gamepad, which works great with the touchscreen used frequently in
+                this game.</p>
+
+            <h4>MelonDS setup</h4>
+            <b>Requirements:</b>
+            <ul>
+                <li>Windows, macOS or Linux computer</li>
+                <li>Nintendo DS or DS lite console</li>
+                <li>Flash cartridge (e.g. R4 cartridge. Also, an SD card to load stuff onto it, if necessary)</li>
```

Comment:
Don't most flash carts use microSD rather than regular SD? should probably specify that.

Reply:
Yeah, it varies, but yeah.

Reply:
Added to the requirements section, but I'd rather not be overly verbose with specifying "SD or microSD" each of the four million times its mentioned :)